common/views/components/OpeningTimes/OpeningTimes.tsx

```diff
@@ -5,46 +5,42 @@ import {
   collectionVenueId,
   getNameFromCollectionVenue,
 } from '@weco/common/data/hardcoded-ids';
-import { FunctionComponent, ReactElement } from 'react';
+import { FC } from 'react';
 
 type Props = {
   venues: Venue[];
 };
 
-const OpeningTimes: FunctionComponent<Props> = ({
```

Comment:
Although a big diff, this is just removing some unnecessary type signatures and removing the `return` from the body of this component.